resources/views/components/forms/textarea.blade.php

```diff
@@ -56,7 +56,7 @@ class="absolute inset-y-0 right-0 flex items-center h-6 pt-2 pr-2 cursor-pointer
 
         </div>
     @else
-        <textarea {{ $allowTab ? '@keydown.tab=handleKeydown' : '' }} placeholder="{{ $placeholder }}" {{ $attributes->merge(['class' => $defaultClass]) }}
+        <textarea {{ $allowTab ? '@keydown.tab=handleKeydown' : '' }} placeholder="{{ $placeholder }}" spellcheck="{{ $spellcheck ? 'true' : 'false' }}" {{ $attributes->merge(['class' => $defaultClass]) }}
```

Comment:
This syntax on an attribute won't work if spellCheck is not passed in. 

You can see how I changed the allowTab attribute here: https://github.com/coollabsio/coolify/pull/1972/files#diff-629cdfbb57994e3afa73247229f9105f4950a9eba54d4b4b19bca1b4a9f04983L59

Reply:
@grahamhealy20 Can you please fix it?

@w3cj Thank you!

Reply:
Hi @w3cj, @andrasbacsai - just pushed up a change now. Thanks for the pointer! 😄